CHANGELOG.md

```diff
@@ -22,6 +39,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `interpreter::Signature` is now `signature::Decoded`
 - `interpreter::SignatureChecker::check_sig` takes a `&signature::Decoded` instead of a `&[u8]`
 - `signature::HashType` now has private fields, but there are corresponding getter methods
+- `Stack` operations have been renamed (#222)
+  - `top` → `rget`
+  - `swap` → `rswap`
+  - `push_back` → `push`
+  - `empty` → `is_empty`
+  - `size` → `len`
+  - `back` → `last_mut`
+  - `erase` → `rremove`
+  - `insert` → `rinsert`
+  - `end` → removed
```

Comment:
These are all now removed from the public API. Presumably this is a merge conflict resolution mistake.
```suggestion
```